### PR TITLE
Update `Beatmap veto` to allow challenging a veto after a year

### PR DESCRIPTION
**Pull request by Hiviexd**
adds a new procedure that allows mappers to re-initiate mediations after a year

basically, it's agreed upon that a veto can be destructive, alongside mindsets and standards can change after a good while (one notable example is [this](https://osu.ppy.sh/beatmapsets/776437/discussion/-/generalAll#/1723667/7292501)), this went through 2 separate votes (one that's NAT-only, and one that's BN+NAT), and the majority agreed that it's fair to let a veto be challenged after a year.

this also had a [proposal](https://osu.ppy.sh/community/forums/topics/1694007?n=1) on the RC forum, also do note that the bulk of the discussion happened internally for a few months.

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md [hunk 1]
```diff
@@ -46,3 +46,16 @@ In order to re-initiate a mediation which resulted in the dismissal of a veto, a
...
+If a beatmap veto is upheld, the mapper may request another mediation if they meet the following criteria:
+
+- The latest veto mediation has been concluded for at least a year.
+- The beatmap currently has 2 Beatmap Nominators who are willing to nominate it, in case the veto is dismissed later.
+
+To request a new mediation, the mapper must contact a member of the NAT corresponding to the game mode of the vetoed beatmap. Said request should include a link to the initial mediation conclusion post and the names of the 2 BNs who are willing to nominate the beatmap. The NAT will then contact the 2 BNs, and if there are no issues, the NAT will re-initiate the mediation with a new randomly selected jury.
+
+**If the beatmap veto is dismissed**, the 2 BNs mentioned above may nominate the beatmap. If said BNs are no longer willing to nominate, their position as Beatmap Nominators may be questioned as a result.
+
+**If the beatmap veto is upheld**, the beatmap may not be re-nominated until appropriate changes are made that remedy the issues present, and the beatmap may not be eligible for another mediation, unless the criteria mentioned above is fulfilled again.
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md:61]:
```suggestion
**If the beatmap veto is upheld**, the beatmap may not be re-nominated until appropriate changes are made that remedy the issues present, and the beatmap may not be eligible for another mediation unless the above criteria are fulfilled again.
```

File: wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md [hunk 2]
```diff
@@ -46,3 +46,16 @@ In order to re-initiate a mediation which resulted in the dismissal of a veto, a
...
+## Challenging a beatmap veto
+
+If a beatmap veto is upheld, the mapper may request another mediation if they meet the following criteria:
+
+- The latest veto mediation has been concluded for at least a year.
+- The beatmap currently has 2 Beatmap Nominators who are willing to nominate it, in case the veto is dismissed later.
+
+To request a new mediation, the mapper must contact a member of the NAT corresponding to the game mode of the vetoed beatmap. Said request should include a link to the initial mediation conclusion post and the names of the 2 BNs who are willing to nominate the beatmap. The NAT will then contact the 2 BNs, and if there are no issues, the NAT will re-initiate the mediation with a new randomly selected jury.
+
+**If the beatmap veto is dismissed**, the 2 BNs mentioned above may nominate the beatmap. If said BNs are no longer willing to nominate, their position as Beatmap Nominators may be questioned as a result.
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md:59]:
```suggestion
**If the veto is dismissed**, the 2 BNs mentioned above may nominate the beatmap. If said BNs are no longer willing to nominate, their position as Beatmap Nominators may be questioned as a result.
```

**Comment by Walavouchey** [comment]
this is cool and all but can you provide reasoning for why this should be a thing? are issues presented in a veto supposed to be considered "potentially expired" because some amount of time has passed?

**Comment by Hiviexd** [comment]
basically, it's agreed upon that a veto can be destructive, alongside mindsets and standards can change after a good while (one notable example is [this](https://osu.ppy.sh/beatmapsets/776437/discussion/-/generalAll#/1723667/7292501)), this went through 2 separate votes (1 NAT-only, and one BN+NAT), and the majority agreed that it's fair to let a veto be challenged after a year

o yea this also had a [proposal](https://osu.ppy.sh/community/forums/topics/1694007?n=1) on the RC forum, completely forgot to link that sorry, that happened because the bulk of the discussion happened internally for a few months.

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**